components/profile-controller/config/default/kustomization.yaml

```diff
@@ -0,0 +1,49 @@
+# Adds namespace to all resources.
```

Comment:
you may want to add these kustomization files to the new repo for kustomization manifests (kubeflow/manifests)

Reply:
ack, will do later